src/moclojer/specs/moclojer.clj

```diff
@@ -28,8 +29,15 @@
                 (render-template request)))))
 
 (defn generic-handler
-  [response]
+  [response webhook]
   (fn [request]
+    (when webhook
+      (whook/request-after-delay
```

Comment:
```suggestion
      (webhook/request-after-delay
```

Reply:
name conflict
- whook is import
- webhook config